packages/bsky/tests/views/notifications.test.ts

```diff
@@ -497,17 +498,150 @@ describe('notification views', () => {
     expect(results(paginatedAll)).toEqual(results([full.data]))
   })
 
-  it('fails open on clearly bad cursor.', async () => {
-    const { data: notifs } =
-      await agent.api.app.bsky.notification.listNotifications(
-        { cursor: '90210::bafycid' },
-        {
-          headers: await network.serviceHeaders(
-            alice,
-            ids.AppBskyNotificationListNotifications,
-          ),
+  describe('notifications delay', () => {
+    const delay0s = 0
+    const delay3s = 3_000
+    const delay5s = 5_000
+
+    let delayNetwork: TestNetwork
+    let delayAgent: AtpAgent
+    let delaySc: SeedClient
+    let delayAlice: string
+
+    beforeAll(async () => {
+      delayNetwork = await TestNetwork.create({
+        bsky: {
+          notificationsDelayMs: delay3s,
         },
+        dbPostgresSchema: 'bsky_views_notifications',
```

Comment:
These will be the same tables as the rest of above notification tests.  I wonder if that was unintentional, since I see the seed gets played on it again.